material-overrides/assets/stylesheets/tanssi.css

```
@@ -679,24 +679,66 @@ details.function,
 }
 
 /* Feedback widget styling */
-.md-feedback {
-  margin: 2em auto;
-  text-align: center;
+.feedback-actions-container {
+  margin-top: 4rem;
+  display: flex;
+  background-color: var(--raisin-black);
   border: var(--border-width) solid var(--md-accent-fg-color);
-  border-radius: 0.2em;
-  width: 30%;
-  padding: 0.7em;
-  font-family: var(--md-header-font);
-  font-weight: var(--md-header-weight);
+  border-radius: var(--border-radius-card);
+  padding: 1.5em;
+  width: calc(100% - 2em);
+  flex-direction: column;
+}
+
+.md-feedback {
+  margin-top: 0;
+  margin-bottom: 0;
+  justify-content: center;
+  display: flex;
 }
 
 .md-feedback__title {
-  font-weight: 300;
+  font-weight: 400;
+  font-size: 12px;
+  margin: 0;
 }
 
-.md-feedback__note>* {
-  font-size: .9em;
-  font-weight: 600;
+.md-feedback__note > * {
+  font-size: 12px;
+  font-weight: 400;
+  font-family: 'Inter';
+}
+
+.md-feedback__list {
+  margin-bottom: 1em;
+  margin-top: 1em;
+  gap: 1em;
+}
+/* Divider Styling */
+.divider {
+  width: 100%;
+  height: 1px;
+  background-color: var(--button-primary-bg-color);
+  margin: 1em 0;
+}
+/*Edit this page button Styling*/
```

```suggestion

/*Edit this page button Styling*/
```

Make sure there is spacing between each block of css